CMakeLists.txt

```diff
@@ -225,17 +236,17 @@ endif()
 # Functions
 # ==================================================================================================
 function(list_licenses OUTPUT MODULES)
-    file(WRITE ${OUTPUT} "R\"FILAMENT__(\n")
     set(_MODULES ${MODULES} ${ARGN})
     foreach(module ${_MODULES})
+	    file(WRITE ${OUTPUT} "R\"FILAMENT__(\n")
```

Comment:
Why this change?

Reply:
Exactly because issue I've mentioned in description - too large string sequential string literals.